### ISSUE TITLE
nikola check --clean-files finish with status=1 even when no error occurs

### ISSUE THREAD

**Issue by lucacerone** [getnikola/nikola#3269]
Hi, I am not sure whether this is a bug or not (but I think the behaviour is not expected)

I am setting up my blog (locally) on an Ubuntu machine and
I would like to run an alias to clean the output directory from orphaned files and automatically run a script to deploy the newest changes to my website using ftp, something along the lines of

```bash
alias publish='nikola check --clean-files && upload_to_remote' 
```
where `upload_to_remote` is the script I'll use to sync my local `output/` folder with the remote `www` folder (for apache)

The problem is that, even if no problem occurs, `nikola check --clean-files` always returns status 1, as shown below

```bash
(venv) .../www$ nikola check --clean-files
Scanning posts........done!
(venv) .../www$ echo $?
1
```

This behaviour makes it impossible to concatenate commands using the `&&` operator in bash.
Is this intended, or is it a bug?

Thanks a lot for your help with this,

Luca

### Environment

Ubuntu Mate 18.04

**Python Version:** v3.6.8

**Nikola Version:** v8.0.2 (installed from pip)

**Operating System:**

**Comment by Kwpolska** [comment]
Yeah, that’s not intentional behavior. Fixed in commit 85aaf5fba.

**Comment by lucacerone** [comment]
Wow, that was super fast :) thanks a lot for fixing it!
